### PR TITLE
new filters from lavadsp

### PR DESCRIPTION
**Pull request by topi314**
added channelmix & lowpass filter

### CODE REVIEW

**Comment by topi314** [comment]
Also adding lowpass later

**Comment by topi314** [comment]
reminder to myself: add docu

File: build.gradle [hunk 1]
```diff
@@ -25,11 +25,11 @@ buildscript {
 allprojects {
 
     repositories {
+        maven { url 'https://m2.dv8tion.net/releases' }
```
Comment (topi314) [inline build.gradle]: this changes are also in https://github.com/freyacodes/Lavalink/pull/484
depending on which gets merged first someone needs to merge it :)

**Comment by topi314** [comment]
this should be done then
also expect a pr for your client :)

**Comment by freyacodes** [comment]
Have these filters been tested?

**Comment by DaliborTrampota** [comment]
> Have these filters been tested?

yes, he and I tested them

**Pull request merged**